### PR TITLE
increasing line height in the radio input type

### LINKED ISSUE #278: [Time sensitive] Add more space between radio options (at least for mobile phones)
@satra @djarecka @sanuann 

In mobile phones, people have been providing feedback that in the radio option, the responses are too close together, and once you click on one it advances to the next item, so people may scroll back to check if the answer was correct. 

Can we add more space between possible responses in a scale?  

image

It's true, you can try from a mobile phone (go to the section "Mental health": 
https://www.repronim.org/reproschema-ui/#/?url=https://raw.githubusercontent.com/mentalwellbeing/WellBeingStudy/master/protocol/WellBeing_schema

### PR DESCRIPTION
**Pull request by djarecka**
closes #278

@danielmlow -  could you please check if this looks better to you

### CODE REVIEW

**Comment by danielmlow** [comment]
@djarecka yes! Much better! Thank you! 

@djarecka  or @satra please merge.

File: src/components/Inputs/WebRadio/Radio.vue [hunk 1]
```diff
@@ -8,6 +8,7 @@
             :options="options"
             stacked
             plain
+            style="line-height: 2.5em"
```
Comment (sanuann) [inline src/components/Inputs/WebRadio/Radio.vue:11]: @djarecka to follow best practices, you could add the style to a class and use the class name in the html element. you can add this style to the class `text-left` here.

Comment (djarecka) [reply src/components/Inputs/WebRadio/Radio.vue:11]: could you please show me how exactly would you do it? Yesterday I've tried to change the `class` element and failed to see any difference, so must have done something wrong

Comment (danielmlow) [reply src/components/Inputs/WebRadio/Radio.vue:11]: @djarecka last night when I used the commit checksum / commitish it worked, but now it doesn't so I'm guessing your latest change created a bug.

Comment (djarecka) [reply src/components/Inputs/WebRadio/Radio.vue:11]: @danielmlow - but i didn't change anything...

Comment (danielmlow) [reply src/components/Inputs/WebRadio/Radio.vue:11]: @djarecka oh, sorry. It wasn't related to your commit. Disregard

Comment (danielmlow) [reply src/components/Inputs/WebRadio/Radio.vue:11]: @djarecka @sanuann very minor, but I did notice that when `"multipleChoice": true` then there is more space for the reponse labels but it's a bit uneven with the boxes:
image

The `"multipleChoice": false` works fine:
image

**Comment by djarecka** [comment]
@danielmlow - yes, I saw it last night, and wasn't sure why it happens and what is the easiest fix. Will take another look tonight or tomorrow.

**Comment by djarecka** [comment]
@danielmlow - I've changed the way how I add spacing for multichoice. Looks better IMO, but could you please check if the values you're getting in the results are correct.

If this works well, I could also change it for a single choice option

**Comment by danielmlow** [comment]
@djarecka Looks great! thanks!

**Comment by djarecka** [comment]
I've updated the radio input type (single choice). If everything looks good, it can be merged

**Comment by danielmlow** [comment]
@djarecka it looks great! I checked I can receive responses correctly. Thanks!

**Pull request merged**